angular-http-server.js

```diff
@@ -23,11 +23,10 @@ var server = http.createServer(function (req, res) {
 
             console.log("Sending file: %s", possibleFilename);
             fileBuffer = fs.readFileSync(possibleFilename);
-            res.writeHead(200, { 'Content-Type': mime.lookup(fileExtension) });
-            // if (fileExtension == "css") {
-            // } else {
-            //     res.writeHead(200, { 'Content-Type': 'application/x-font-eot' });
-            // }
+            res.writeHead(200, { 
```

Comment:
@dhdanno I believe this will allow static files to be cached by our application for 5 days